app/views/snippets/typography_warning_text.html

```diff
@@ -1,6 +1,6 @@
 <div class="notice">
   <i class="icon icon-important">
-    <span class="visually-hidden">Warning</span>
+    <span class="visually-hidden">Important</span>
```

Comment:
Should this have a colon?

Reply:
I think the spans and i class space the reading out of the text in most, if not all, cases. I do not think it needs a separator like a colon.

Reply:
I'd rather give it a exclamation mark, that would also let some screen readers add more emphasis. Although Steven mentioned in an email that it "would seem odd for the government to do that". I'm not sure, I think it would be appropriate here.

Reply:
I do not think you need any punctuation. It acts almost like a heading.

Reply:
But it would make more typographic sense. It's "Warning You can be fined up to £5,000 if you don’t register." vs "Warning! You can be fined up to £5,000 if you don’t register."

Reply:
I know some screen readers change inflection and some do not. If the content was visible, it would not have an exclamation mark at the end. It would be separated or have a colon. But I do not think either is needed from a content design point of view.

Reply:
I would disagree that it wouldn't have an exclamation mark if the content was visible. I'm not sure about government guidance on exclamation marks, so it might not be there for style guide reasons. But personally, I would add an exclamation mark if it was visible and have seen lots of (non-government) examples where there was one.

Reply:
There is nothing forbidding them but they are not common in government writing. The Guardian style guide does say "Use sparingly! (As Scott Fitzgerald said, it is like laughing at your own jokes)". If it was visible, it would likely have a colon or a dash, or be a heading.